cmd/az-analyze/cmd/azva.go

```diff
@@ -116,20 +123,15 @@ type AzvaResourceAll struct {
 }
 
 // return azVolumeAttachements with all Pods/Nodes/Zones when no flags is provided
-func GetAllAzVolumeAttachements(namespace string) []AzvaResourceAll {
-	// access to config and Clientsets
-	config := getConfig()
-	clientsetK8s := getKubernetesClientset(config)
-	clientsetAzDisk := getAzDiskClientset(config)
-
+func GetAllAzVolumeAttachements(clientsetK8s kubernetes.Interface, clientsetAzDisk versioned.Interface, namespace string) []AzvaResourceAll {
 	result := make([]AzvaResourceAll, 0)
 
 	if namespace == "" {
 		namespace = "default"
 	}
 
 	// get pvc claim names of pod(s)
-	pvcClaimNameSet := make(map[string]string)
+	pvcClaimNameSet := make(map[string][]string)
```

Comment:
Some of the code for  GetAllAzVolumeattachments and GetAzVolumeAttachmentsByPod seems same. Would it be possible to take that part out in a function and resuse that function in both the GetAllAzVolumeattachments and  GetAzVolumeAttachmentsByPod .